astroid/interpreter/objectmodel.py

```diff
@@ -799,7 +808,9 @@ def attr_fset(self):
 
         func = self._instance
 
-        def find_setter(func: objects.Property) -> Optional[astroid.FunctionDef]:
+        def find_setter(
+            func: Property,  # pylint: disable=used-before-assignment
+        ) -> Optional[astroid.FunctionDef]:
```

Comment:
```suggestion
        def find_setter(func: "Property") -> Optional[astroid.FunctionDef]:
```
This current code causes a runtime exception as `Property` isn't defined. Use `"Property"` instead.